source/includes/fact-legacy-connections.rst

```diff
@@ -0,0 +1,8 @@
+Starting in version 1.39.2, |compass| no longer supports  migration from legacy 
+connection files that pre-date version 1.31.0. Legacy connections refer to an 
+internal |compass-short| connection-options format that is stored on disk and no 
+longer supported beyond |compass| version 1.39.0. 
```

Comment:
[clarity]

Maybe replace "beyond" with "after" ?

Or alternatively, "no longer supported starting in version 1.39.2", whichever you think is clearer.